src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const list = document.querySelectorAll('.population');
+const arr = [...list];
+let total = 0;
+let count = 0;
+const validArr = [];
+
+for (let i = 0; i < arr.length; i++) {
+  validArr.push([]);
```

Comment:
you can store the result of your calculation without saving temporary data like this validArr (which by the way a bad name)